src/lab_sim/config/control/picknik_ur.ros2_control.yaml

```
@@ -213,17 +223,34 @@ velocity_force_controller:
       - 0.524
       - 0.524
       - 0.524
+    # Maximum Cartesian-space velocities.
     max_cartesian_velocity:
       - 0.25
       - 0.25
       - 0.25
       - 1.5707
       - 1.5707
       - 1.5707
+    # Maximum Cartesian-space accelerations.
     max_cartesian_acceleration:
       - 2.0
       - 2.0
       - 2.0
       - 4.0
       - 4.0
       - 4.0
+    # Additional optional parameters for reference (set to defaults):
+    # Cutoff frequency ratio for the FTS filter. Valid values range from 0 to 1, where 1 is the sampling frequency.
+    ft_cutoff_frequency_ratio: 1.0
+    # Rate in Hz at which the controller will publish the state.
+    state_publish_rate: 50.0
```

I think the generate_parameters_library freaks out if it gets a double when it expected an int.
```suggestion
    state_publish_rate: 50
```